internal/pkg/db/memory/export_test.go

```diff
@@ -4,24 +4,22 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
-package clients
+package memory
```

Comment:
This test should live in edgex-go/internal/pkg/db/test
You can look how it has been done for data and metadata. This way we can reuse this test for all the db implementations.